code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -195,15 +187,6 @@
 	..()
 	gear_tweaks = list(gear_tweak_free_color_choice)
 
-/datum/gear/head/loose_beanie
```

Comment:
Not sure what is wrong with this item in particular?

Reply:
Poorly sprited.